### PR TITLE
feat(courrier-type): add page and search for standard mail

### PR DESCRIPTION
**Pull request by lionelB**
Ajout des modele de courriers 
capture d ecran 2018-10-09 a 10 04 47

capture d ecran 2018-10-09 a 10 03 25

### CODE REVIEW

**Comment by revolunet** [comment]
perfect merci !

2 petites notes : 
 - message "Article introuvable" -> "Modèle de courrier introuvable"
 - Titre "Code du travail : xxx" -> "Modèle de lettre : xxx"

**Review by revolunet — APPROVED** [review]

File: packages/code-du-travail-api/src/server/index.js [hunk 1]
```diff
@@ -22,6 +24,17 @@ app.use(cors())
 app.use(bodyParser())
 app.use(apiRoutes.routes())
 
+// Handle file download
+apiRoutes.get(`${API_BASE_URL}/docs/:filename`, async ctx => {
```
Comment (revolunet) [inline packages/code-du-travail-api/src/server/index.js]: ⚠️ `/api/v1/docs/..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2Fetc..%2Fpasswd`

Comment (lionelB) [reply packages/code-du-travail-api/src/server/index.js]: hug

updated !

**Pull request merged**